src/inc/gcinfotypes.h

```diff
@@ -240,6 +240,25 @@ inline bool IsValidReturnKind(ReturnKind returnKind)
         ;
 }
 
+inline bool IsGCReturnKind(ReturnKind returnKind)
+{
+    switch (returnKind)
+    {
+    case RT_Object:
```

Comment:
When we protect the value using `FrameWithCookie<GCFrame> gcFrame;` later, we always assume that the return value is `RT_Object`. Are all the other cases except `RT_Object`  broken?